<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanas</class>
 <widget class="QMainWindow" name="ventanas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Game_window" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>681</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>651</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="L_nivel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nivel</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_vidas">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>vidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_puntos">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>puntos</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
